cluster_experiments/power_analysis.py

```diff
@@ -115,6 +119,10 @@ def __init__(
             target_col=target_col,
             features_cupac_model=features_cupac_model,
         )
+        if seed:
```

Comment:
```suggestion
        if seed is not None:
```

perhaps an edge case, but if seed = 0, won't seed anything

Reply:
Good catch! I changed it to your suggestion